### ISSUE TITLE
[POEM] Cannot run the pre-trained checkpoint model

### ISSUE THREAD

**Issue by rahag** [google-research/google-research#841]
Hi,

I tried running infer.py in pr_vipe using the checkpoint provided here. [https://sites.google.com/view/pr-vipe]

I get this error.

`tensorflow.python.framework.errors_impl.DataLossError:` Unable to open table file checkpoint_path\model.ckpt-02013963.data-00000-of-00001: Data loss: not an sstable (bad magic number): perhaps your file is in a different file format and you need to use a different restore operator?`

I am not sure if I am giving the wrong input file or some wrong arguments.

**Comment by HakuWang** [comment]
Hi,  i also tried running infer.py using the pre-trained model but could not get the input flags work. 

The flags i input is 

--input_csv "inputs\sample_input.csv" --output_dir "outputs\" --checkpoint_path "model.ckpt-02013963"

My file structure like this:
![无标题](https://user-images.githubusercontent.com/35250931/192929783-342008b3-9c63-4cf7-8c93-b2f8c9a72dbd.jpg)
